packages/vuetify/src/framework.ts

```diff
@@ -14,59 +19,64 @@ import {
 // Services
 import * as services from './services'
 
-export default class Vuetify {
-  static install = install
+export const VuetifySymbol = Symbol.for('vuetify')
```

Comment:
Use `InjectionKey` from vue here.

```ts
export const VuetifySymbol: InjectionKey<Framework> = Symbol.for('vuetify')
```

That way `useVuetify` gets a proper return type